webapp/src/components/expanded_view/component.tsx

```diff
@@ -737,27 +862,66 @@ export default class ExpandedView extends React.PureComponent<Props, State> {
                     </div>
                 </div>
                 { this.state.showParticipantsList &&
-                <ul style={style.rhs as CSSProperties}>
+                <ul style={styles.rhs}>
                     <span style={{position: 'sticky', top: '0', background: 'inherit', fontWeight: 600, padding: '8px'}}>{'Participants list'}</span>
                     { this.renderParticipantsRHSList() }
                 </ul>
                 }
+                {globalRhsSupported && <ExpandedViewGlobalsStyle callThreadSelected={this.props.rhsSelectedThreadID === this.props.threadID}/>}
             </div>
         );
     }
 }
 
-const style = {
+const isActiveElementInteractable = () => {
+    return document.activeElement && ['INPUT', 'SELECT', 'BUTTON', 'TEXTAREA'].includes(document.activeElement.tagName);
+};
+
+const UnreadIndicator = ({mentions}: {mentions?: number}) => {
+    return (
+        <UnreadDot>{mentions && mentions > 99 ? '99+' : mentions || null}</UnreadDot>
```

Comment:
I didn't realize the number should be there: 
<img width="130" alt="image" src="https://user-images.githubusercontent.com/1490756/199542990-da9fb619-50f4-4571-8dd7-56a979fdc3b9.png">

Reply:
Yep, for unread mentions.

Reply:
Ah, gotcha. Looks very nice!